.github/workflows/build-image.yml

```
@@ -57,9 +57,15 @@ jobs:
             type=pep440,pattern=v{{major}}.{{minor}}
             type=ref,event=pr
 
+      - name: Set version suffix to +edge-<sha>
+        id: git_vars
+        run: echo "MASTODON_VERSION_SUFFIX=+edge-$(git rev-parse --short HEAD)" >> $GITHUB_ENV
```

Should the value string here be quoted? I know both parse, but I've always quoted the values, so:
```suggestion
        run: echo "MASTODON_VERSION_SUFFIX=\"+edge-$(git rev-parse --short HEAD)\"" >> $GITHUB_ENV
```

---

👍